### DISCUSSION TITLE
Can't login

### DISCUSSION THREAD

**Discussion by aardbol** [j-fbriere/squawker#294] [Q&A] [answered]
I'm trying to login with my account via the settings, but it never works. I used my username and password. How can I solve this?

**Comment by j-fbriere** [comment]
What happens when you tap on "Add an account" form Save button?
Do you have an error? Please show the error.

**Comment by aardbol** [comment]
![Screenshot_20240430-215413_1](https://github.com/j-fbriere/squawker/assets/14614620/191e3d33-4100-4e6c-a0a1-2562e830b80a)

**Reply by aardbol** [reply]
This happens when I enter username and password

**Comment by j-fbriere** [comment]
Are you able to login from the Twitter/X site in a browser with the same username password?

**Reply by aardbol** [reply, marked as answer]
Yeah, but it was caused by having (hardware token) 2FA enabled. Temp disabling it solved it
